### ISSUE TITLE
Performance of external svg on browsers

### ISSUE THREAD

**Issue by SLQ-CommWeb** [maxwellito/vivus#43]
I know that you said in your notes that the performance on older versions of Firefox does not perform to well but I have noticed that the svg does not animate at all on the latest version of Firefox (Mac and PC) and just displays a static SVG file. This happens when using an external SVG and not doing it inline. Using the SVG tags inline in the HTML document works absolutely fine.

Also it's causing IE11 browser to pretty much hang for a few minutes and then display a static SVG at the end as well (never been a big fan of IE but we have to support it).

It would be nice to work externally on the browsers as it would be good to avoid writing all the SVG code inline.

**Comment by maxwellito** [comment]
For IE, I'm not proud of the trick I used to make it working which is to recreate the DOM at every update to force the rendering. It's a huuuuge waste of performances but the only way to make the animation working. Otherwise it's pure glitch, yark!

About Firefox, my bad, I admit I haven't tried this new feature (which has been released last week) on Firefox, let me try that.

I tag the issue as a bug, thanks for the spot. I come back to you once I have more details.
